app/sync.js

```diff
@@ -307,20 +309,19 @@ module.exports.onSyncReady = (isFirstRun, e) => {
  */
 module.exports.init = function (appState) {
   const initialState = appState.get('sync') || new Immutable.Map()
-  const RELOAD_MESSAGE = 'reload-sync-extension'
-  const RESET_SYNC = 'reset-sync'
   const reset = () => {
     log('Resetting browser local sync state.')
     appActions.changeSetting(settings.SYNC_ENABLED, false)
     appActions.changeSetting(settings.SYNC_DEVICE_NAME, undefined)
     appActions.resetSyncData()
   }
   // sent by about:preferences when sync should be reloaded
-  ipcMain.on(RELOAD_MESSAGE, () => {
-    process.emit(RELOAD_MESSAGE)
+  ipcMain.on(messages.RELOAD_SYNC_EXTENSION, () => {
```

Comment:
`messages` here is `js/constants/sync/messages`, but it should actually be `js/constants/messages`